### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by samanpwbb**
@elifitch this branch is a fork off of the react-16 branch you were working on a while back. Did we have a backlog of things we wanted to take care of before the 1.0.0 release? Going to create a new 1.0.0 prerelease now to unblock https://github.com/mapbox/studio/pull/14704

cc: @thefifthisa

### CODE REVIEW

**Comment by samanpwbb** [comment]
@katydecorah do you think you could give this a review when you get a chance? Studio is reliant on this branch. I don't see a good reason not to go 1.0.0 now. We can always go to 2.0.0 when we need it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.0.0-beta.2
+
+- [fix] Fix coloring values for popover
+## 1.0.0-beta.0
+- [feature] Trap events in modals by default
+- [fix] rename unsafe lifecycle methods to eliminate warnings under React 16
```
Comment (katydecorah) [inline CHANGELOG.md:8]: - Are we waiting until we upgrade to React 17 to remove these unsafe lifecycle methods? Is it out of scope to do it now?
- Is the goal is to eliminate all lifecycle warnings? If so, we will want to rename `componentWillMount` to `UNSAFE_componentWillMount`.
- Likely out of scope, but `react-datepicker` is on an older version that triggers unsafe lifecycle methods. We could update to eliminate the warnings generated from this pacakge.

**Review by katydecorah — COMMENTED** [review]
Lookin good! I have a few questions, see line comments:

File: package.json [hunk 2]
```diff
@@ -98,9 +98,9 @@
     "pify": "^3.0.0",
     "prettier": "^2.3.1",
     "prismjs": "^1.15.0",
-    "react": "^16.4.1",
+    "react": "^16.13.1",
     "react-docgen": "^2.21.0",
-    "react-dom": "^16.4.1",
+    "react-dom": "^16.13.1",
```
[the remark this answers is not in the document]
Comment (samanpwbb) [reply package.json:103]: Yep! I will add a commit.

File: CHANGELOG.md (hunk 1 above)
Comment (samanpwbb) [reply CHANGELOG.md:8]: Yeah, let's finish the of updating our lifecycle methods. I can take care of it.

**Pull request merged**